rtrlib/lib/vector.c

```diff
@@ -0,0 +1,154 @@
+
+
+#include "vector.h"
+
+#include <math.h>
+
+int aspa_vector_create(struct aspa_vector **vector_pointer)
+{
+	const size_t DEFAULT_INITIAL_SIZE = 128;
```

Comment:
shouldn't that be in a `#define`?

Reply:
might consider const enums too, as 12) in https://www.kernel.org/doc/html/v4.10/process/coding-style.html

Reply:
From a compiler standpoint, this absolutely doesn't matter, the const values will be compiled in any way. In addition, this probably more aesthetic and more typesafe.